app/templates/components/hero-desktop.html

```diff
@@ -0,0 +1,66 @@
+<div
+    class="hero-section hidden fixed top-0 left-0 bg-corporate-blue text-white h-screen max-w-hero overflow-hidden desktop:flex"
+>
+    <div
+        id="slider"
+        class="w-2x-hero h-full translate-x-0 flex ease-in duration-300 {% if false %}-translate-x-1/2{% endif %}"
+    >
+        <div class="left h-full w-hero flex flex-col justify-between">
+            <div
+                class="w-full px-desktop max-w-4xl pt-10 mx-auto flex flex-col justify-start"
+            >
+                <img
+                    class="w-44 self-end desktop:mb-16"
+                    src="{{ url_for('static', path='images/TSB-Logo-outline-weiss.svg') }}"
+                    alt="{{ translations.hero.imageAlt }}"
+                />
+                <div class="w-80 self-center">
+                    <h1 class="pt-4 pb-2 font-bold">
```

Comment:
The headlines are a bit "agile" right now – this is how they look:

![Bildschirmfoto 2023-09-27 um 11 43 30](https://github.com/technologiestiftung/innovationserhebung/assets/6429568/9f4128dd-139f-4a9a-86bc-6507e55c5c39)


- There should only be one `<h1>`
- All the other headlines should be nested without skipping any intermediate levels
- There should be no duplicates

I guess, ideally it should look something like this in the end:

```
H1: Innovationsmonitor der Stadt Berlin
  H2: Erhebung 1
  H2: Erhebung 2
  H2: Erhebung 3
```